### PR TITLE
prints wallet address with pk mnemonic when using 'generate and print'

### PR DESCRIPTION
**Pull request by sigmarelax**
Adding the first wallet address to the terminal output for 'chia keys generate_and_print' is practical for many use-cases. Cold storage wallets can now be outputted and used for storing xch without ever writing the private key to disk

### CODE REVIEW

**Review by mariano54 — DISMISSED** [review]
Thank you!

File: chia/cmds/keys_funcs.py [hunk 1]
```diff
@@ -7,20 +7,31 @@
...
     """
-    Generates a seed for a private key, and prints the mnemonic to the terminal.
+    Generates a seed for a private key, then prints the mnemonic and first wallet address to the terminal.
     """
 
     mnemonic = generate_mnemonic()
     print("Generating private key. Mnemonic (24 secret words):")
     print(mnemonic)
+    passphrase = ""
+    root_path = DEFAULT_ROOT_PATH
```
Comment (Nikolaj-K) [inline chia/cmds/keys_funcs.py:25]: Seems like you can directly pass the `DEFAULT_ROOT_PATH` one line below.

File: chia/cmds/keys_funcs.py [hunk 2]
```diff
@@ -7,20 +7,31 @@
...
 
 
 def generate_and_print():
     """
-    Generates a seed for a private key, and prints the mnemonic to the terminal.
+    Generates a seed for a private key, then prints the mnemonic and first wallet address to the terminal.
     """
 
     mnemonic = generate_mnemonic()
     print("Generating private key. Mnemonic (24 secret words):")
```
Comment (Nikolaj-K) [inline chia/cmds/keys_funcs.py:22]:
```python
print(f"Generating private key. Mnemonic (24 secret words): {mnemonic}")
```

File: chia/cmds/keys_funcs.py [hunk 3]
```diff
@@ -7,20 +7,31 @@
...
+    passphrase = ""
+    root_path = DEFAULT_ROOT_PATH
+    config = load_config(root_path, "config.yaml")
+    selected = config["selected_network"]
+    prefix = config["network_overrides"]["config"][selected]["address_prefix"]
+    seed = mnemonic_to_seed(mnemonic, passphrase)
+    sk = AugSchemeMPL.key_gen(seed)
+    print(
+            "First wallet address:",
+            encode_puzzle_hash(create_puzzlehash_for_pk(master_sk_to_wallet_sk(sk, uint32(0)).get_g1()), prefix),
```
Comment (Nikolaj-K) [inline chia/cmds/keys_funcs.py:33]: I wouldn't do a computation "this big" inside the print statement.

**Review by wjblanke — DISMISSED** [review]
aok

File: chia/cmds/keys_funcs.py [hunk 4]
```diff
@@ -22,12 +23,23 @@
 
 def generate_and_print():
```
Comment (jack60612) [inline chia/cmds/keys_funcs.py:24]: add root_dir here, ex: `def generate_and_print(chia_root: Path)`

**Review by jack60612 — CHANGES_REQUESTED** [review]
These are some minor changes that will help your pr align with the rest of our repository better. contact me on keybase if you have any questions, I'm always happy to help.

**Comment by jack60612** [comment]
Ill make the changes i requested into this pr later

**Pull request closed without merging**